### ISSUE TITLE
Can't Open localhost:3000

### ISSUE THREAD

**Issue by iamsado** [zeit/next.js#12331]
# Bug report
When I try to open localhost I gave a error which Module not found: Can't resolve 'styles.css' in '/Users/sado/demo-2/pages'

## Describe the bug
When I try to open localhost I gave a error which "Module not found: Can't resolve 'styles.css' in '/Users/sado/demo-2/pages'"

I created "_app.js" file  in pages folder then I created a css  file styles.css
when I  write "npm run dev" on terminal I gave a error:
[ wait ]  starting the development server ...
[ info ]  waiting on http://localhost:3000 ...
[ error ] ./pages/_app.js
Module not found: Can't resolve '../styles.css' in '/Users/sado/demo-2/pages'
How can I solve this problem?

PLEASE HELP ME!

## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:

## Expected behavior
Successful opening of localhost.

## Screenshots
<img width="682" alt="Ekran Resmi 2020-04-30 01 41 00" src="https://user-images.githubusercontent.com/64561601/80653849-b9c0f180-8a83-11ea-9e1d-e9f06bdfea52.png">
<img width="1552" alt="Ekran Resmi 2020-04-30 01 41 09" src="https://user-images.githubusercontent.com/64561601/80653856-be85a580-8a83-11ea-81ee-1cc47233f03a.png">

## System information

- OS: macOS
- Browser: Chrome
- Version of Next.js: [e.g. 6.0.2]
- Version of Node.js: [e.g. 10.10.0]

**Comment by c0b41** [comment]
remove style.css in pages folder and then call for another directory.

**Comment by iamsado** [comment]
> remove style.css in pages folder and then call for another directory.

Thank you so much.  i really appreciate your help. Why couldn't I think of it.

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
